build.sbt

```diff
@@ -312,15 +335,36 @@ lazy val stagedRelease = (project in file("core/src/test"))
 
 import ReleaseTransformations._
 
-// Don't use sbt-release's cross facility	
+// Don't use sbt-release's cross facility
 releaseCrossBuild := false
 
-def crossReleaseStep(step: ReleaseStep): Seq[ReleaseStep] = {
+lazy val downVersionPySpark = taskKey[Unit]("Replace PySpark")
```

Comment:
To future-proof this for PySpark 3.0.1+ and avoid hard-coding this, I think it'd be useful to change this to "changePysparkVersion" and accept a parameter so we can toggle versions more freely without adding a bespoke task per version.

Reply:
I parametrized it to sparkOld. Assumption is that the environment.yml file has the sparkNew version of pyspark.